docs/hadoop-catalog.md

```diff
@@ -25,19 +25,30 @@ Hadoop 3. If there's any compatibility issue, please create an [issue](https://g
+| `s3-secret-access-key`                             | The secret key of the AWS s3.                                                                                                                                                                                                                                                                                 | (none)          | Yes if use s3                                               | 0.7.0-incubating |
+| `gcs-service-account-file`                         | The path of GCS service account JSON file.                                                                                                                                                                                                                                                                    | (none)          | Yes if use gcs                                              | 0.7.0-incubating |
+| `oss-endpoint`                                     | The endpoint of the Aliyun oss.                                                                                                                                                                                                                                                                               | (none)          | Yes if use oss                                              | 0.7.0-incubating |
+| `oss-access-key-id`                                | The access key of the Aliyun oss.                                                                                                                                                                                                                                                                             | (none)          | Yes if use oss                                              | 0.7.0-incubating |
+| `oss-secret-access-key`                            | The secret key of the Aliyun oss.                                                                                                                                                                                                                                                                             | (none)          | Yes if use oss                                              | 0.7.0-incubating | 
+
+:::warning
+Since Gravitino 0.7.0-incubating, the Hadoop catalog supports S3, GCS, and OSS. To enable the support for these object storages, you need to put the corresponding bundle jar like `gravitino-aws-bundle-{version}.jar`, `gravitino-gcs-bundle-{version}.jar`, `gravitino-oss-bundle-{version}.jar`
+into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory and set the corresponding configurations.
+:::
```

Comment:
Can you please separate the FS support into different table with different headings, like HDFS, S3, xxx.

Reply:
I was hesitate to add it separately as some properties like `filesystem-providers` are shared by several cloud storage, what about adding the following content

```markdown

### Properties required to create a local file system Hadoop Catalog

No additional properties are required to create a local file system Hadoop catalog. 

### Properties required to create an HDFS Hadoop Catalog

Similarly, no additional properties are required to create an HDFS Hadoop catalog if the target HDFS cluster is not Kerberos-enabled. If the target HDFS cluster is Kerberos-enabled, the following properties are required:
- `authentication.type`: The type of authentication for the Hadoop catalog. The value should be `kerberos`.
- `authentication.kerberos.principal`: The principal of the Kerberos authentication.
- `authentication.kerberos.keytab-uri`: The URI of the keytab for the Kerberos authentication.

### Properties required to create an S3 Hadoop Catalog
To create an S3 Hadoop catalog, you need to set the following properties:
- `filesystem-providers`: The value should be `s3` or a comma-separated list of that contains `s3` like `builtin-local,s3`. 
- `s3-endpoint`: The endpoint of the AWS S3.
- `s3-access-key-id`: The access key of the AWS S3.
- `s3-secret-access-key`: The secret key of the AWS S3.

At the same time, you need to put the corresponding bundle jar `gravitino-aws-bundle-{version}.jar` into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory.

### Properties required to create a GCS Hadoop Catalog
To create a GCS Hadoop catalog, you need to set the following properties:
- `filesystem-providers`: The value should be `gcs` or a comma-separated list of that contains `gcs` like `builtin-local,gcs`.
- `gcs-service-account-file`: The path of the GCS service account JSON file.

At the same time, you need to put the corresponding bundle jar `gravitino-gcs-bundle-{version}.jar` into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory.

### Properties required to create an OSS Hadoop Catalog
To create an OSS Hadoop catalog, you need to set the following properties:
- `filesystem-providers`: The value should be `oss` or a comma-separated list of that contains `oss` like `builtin-local,oss`.
- `oss-endpoint`: The endpoint of the Aliyun OSS.
- `oss-access-key-id`: The access key of the Aliyun OSS.
- `oss-secret-access-key`: The secret key of the Aliyun OSS.

At the same time, you need to put the corresponding bundle jar `gravitino-oss-bundle-{version}.jar` into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory.
```

Reply:
Updated.